core/src/App.html

```diff
@@ -30,7 +46,20 @@
         handleRouteChange(trimLeadingSlash(window.location.pathname), component, node, config);
     };
 
-    const handleNavigation = async (component, data) => {
+    const handlePreserveView = (component, data, config, node) => {
```

Comment:
maybe a better name is _addPreserveView_, to make it clear this function does not apply when we navigate back to a preserved view and we "remove" it from the preserved views

Reply:
done